tools/nightly-unit-tests/src/bitrise_yml_writter.py

```diff
@@ -91,6 +91,11 @@ def __rendered_template(self):
 
         template = template.replace('## <MACOS VERSION> ##', self.host_os_version)
 
+        if self.host_os_version.startswith('11'): # macOS 11.X Big Sur
+            template = template.replace('## <RETRY_ON_FAIL> ##', "test_repetition_mode: 'retry_on_failure'")
```

Comment:
Don't we need to specify `maximum_test_repetitions` as well? Or does it have a default value?

Reply:
yes, default value is 3, reasonable i guess.?
i didn't want to replace multiple lines in yaml in order not to deal with indentation issues.

Reply:
Totally 👍 for keeping it simple like that.